ElectronClient/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -756,6 +762,7 @@ const TinyMCE = (props:NoteBodyEditorProps, ref:any) => {
 
 		editor.on('keyup', onKeyUp);
 		editor.on('keypress', onKeypress);
+		editor.on('compositionend', onCompositionEnd);
```

Comment:
Could you attach the event directly to onChangeHandler, like for the cut event for example?

Reply:
Done.